giddy-banana/services/articleService.js

```diff
@@ -0,0 +1,76 @@
+import db from "../models/index.cjs";
+import { NotFoundException } from "./serviceErrors.js";
+
+
+
+/**
+ * Returns information about a user.
+ *
+ * @param {string} _title the article title
+ * @return {Object} JSON with the requested article's data
+ * @throws {Error} If the article couldn't be found or the database
+ * connection was refused.
+ */
+
+async function getArticle(_title) {
```

Comment:
Regarding the parameter naming - shall we stick to what I already started doing? Which is no underscore in the parameter names.

Reply:
Ok, sure. I'll change it.